From c12d1681404ac07b57de8a9bf94b81ba244a9269 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 29 May 2019 16:21:59 +0300
Subject: [PATCH] [SW-4185] gaudi: move the code to scratch pad SRAM

This is a part if the W/A for the PLL clock switch h/w bug.
We move the preboot in to scratch pad SRAM, so there will be no
transactions on the HBW (e.g. instruction fetch) during the PCI PLL HBW
clock switch.

Change-Id: I5c2deff5e113b4d857f8d48965394e61d111dee8
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/preboot.lds b/gaudi/preboot.lds
index ea1f531..01774e2 100644
--- a/gaudi/preboot.lds
+++ b/gaudi/preboot.lds
@@ -5,7 +5,7 @@
  * SPDX-License-Identifier:     GPL-2.0+
  */
 
-MEMORY { .sram : ORIGIN = 0x7ff0000000,
+MEMORY { .sram : ORIGIN = 0x7FFBFE0000,
 		LENGTH = 0x10000 }
 
 OUTPUT_FORMAT("elf64-littleaarch64", "elf64-littleaarch64", "elf64-littleaarch64")
